backend/email_body.md

```diff
@@ -1,44 +1,79 @@
 
-If you are driving there are parking facilities on site.
+The nearest underground stations are:
 
-### Accessibility
+Euston Square
+Euston
+Warren Street 
+King’s Cross St Pancras
+Euston Square has step-free access between the platform and street level from the Westbound platform only.
 
-Accessibility information can be found on the [Edgbaston](https://edgbaston.com/events/) website and at [accessable.co.uk](https://www.accessable.co.uk/birmingham-city-council/access-guides/edgbaston-cricket-club#22525f31-f8b4-407c-b9b7-ea441da91e90).
+Find out how to get to us by tube with Transport for [London’s journey planner](https://tfl.gov.uk/plan-a-journey/).
 
-Please note that we have not been involved in the verification of the information recorded on the websites listed above.
+#### TRAIN
 
-### Additional Conference Related Activities
+The nearest mainline stations are:
 
-A post-conference ‘speed dating’ inspired networking session will be hosted at Edgbaston Stadium (from 17:15 until 18:00) to provide community member’s with an opportunity to be introduced to other event attendees in the afternoon of Tuesday 17<sup>th</sup> October 2023 immediately following the last session of the day.
+Euston
+King’s Cross
+St Pancras International
 
-A post-conference social in the evening of Tuesday 17<sup>th</sup> October 2023 (from 20:00 till late) at [NQ64](https://nq64.co.uk/birmingham/), which is a short walk from the city centre. If you want to know more details, speak to one of the team at check-in and they can let you know.
+#### BUS
+
+The following buses stop close to our main entrance at 215 Euston Road:
+
+18, 30, 73, 205, 390
+
+Find out how to get to us by bus with Transport for [London’s journey planner](https://tfl.gov.uk/plan-a-journey/).
+
+#### BIKE
+
+There are bike stands near the main entrance of Wellcome Collection on Euston Road, and at the side of the building on Gordon Street. 
+
+There are also [Santander docking stations](https://tfl.gov.uk/modes/cycling/santander-cycles) behind the building on Gower Place. 
+
+#### CAR
+
+If you’re using a taxi, there are drop off points by our:
+
+main visitor entrance at 215 Euston Road (raised kerb)
+staff-only entrance on Gower Place (raised and dropped kerb). It’s a short distance from here to our main visitor entrance at the front of the building.
+We have two parking spaces for [Blue Badge holders](https://wellcome.org/about-us/contact-us#accessibility). 
+
+#### Reserve a seat!
+
+If you would like us to reserve you a seat in the main auditorium please email us at [nhs.rcommunity\@nhs.net](mailto:nhs.rcommunity@nhs.net).
+
+For further advice on travel, accessibility and venue information please see the [main conference website](https://nhsrcommunity.com/conference.html)
+
+### Social Programme
+
+We have opted not to arrange any formal social events this year, but invite anyone to post on the #conference Slack channel if you want to propose a meet up at a particular place and time.
```

Comment:
as we suggest something immediately below, perhaps remove this sentence or change next one to state that it's not a formally organised social.